corona-warn-app/cwa-app-android
# Add condition for loading test result (EXPOSUREAPP-12725)

The test result should be only loaded if the consent for submitting is given, so this condition is added.

How to test:

- Scan a positive rat or pcr test (e.g. cwa rat gen -e wru)
- Register for me
- In first consent screen tap "Accept" Button
- In the next screen tap "Consent warn others granted" button
- Switch Toggle Button "Warn others" to Stopped and go back
- Then tap next in "Your test result available" screen
- Close x "Your test result" screen
- In the upcoming popup choose "Do not warn others"
- On the main fragment you will see the positive test result then
- When navigating to main fragment from background or from another tab, you will see the test will not show the reload animation anymore
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12725
